test/build-images.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+
+: ${1?"Need to pass the CLI channel for which to build the Docker images"}
```

Comment:
Can't this be discovered from within this script so that as channels are added, the netci.groovy script doesn't have to be updated each time?